### ISSUE TITLE
Cannot access the radio buttons allowing you to specify whether or not you want to store files larger than 4 gb with a screen reader while creating hidden volumes on disks.

### ISSUE THREAD

**Issue by Tetrismonster** [veracrypt/VeraCrypt#1042]
Hey.
I'll provide the steps I used that should hopefully recreate this issue.
1: Use a screen reader like nvda. Ensure you use tab and shift tab to navigate the volume creation wizard.
2: I used a usb flash drive to do this, so that might be your best bet.
Select to encrypt the entire disk and create a hidden volume.
3: Create the outer volume as you normally would, everything should work fine.
4: When you specify that you want to store files larger than 4 gb in the hidden volume, pressing tab and shift tab won't be able to bring the yes and no options or the next button into focuss, the mouse is required for this.
These buttons aren't being detected by the screen reader.
Let me know if you need any other information.
A few other oddities that occur when using a screen reader.
The screen reader isn't able to automatically read the steps when creating volumes. You are required to alt tab out of the window and back in again.
The screen reader doesn't detect the progress bar during formatting of file containers and so we have no idea whether or not the format is working or how far along it is.
If entering passwords that are invalid or that do not match, the next button simply doesn't appear to the screen reader, and neither does a prompt telling what the issue is.
Lastly, the bootloader has no auditory feedback for anything.
I understand why this is, but would it be possible to create a few beeps and boops to let us know of the following:
1. When the bootloader has booted successfully.
2. When to enter our passwords.
3. A tone to indicate the registering of a key press.
4. A tone telling us when our password is incorrect/the disk can't be mounted.
5. A tone telling us when the disk is mounted.
Thanks in advance.
